pybossa/api/project.py

```diff
@@ -51,6 +51,16 @@ class ProjectAPI(APIBase):
     private_keys = set(['secret_key'])
     restricted_keys = set()
 
+    def _preprocess_request(self, request):
+        # Limit maximum post data size.
+        content_length = request.content_length if request else 0
+        max_length_mb = current_app.config.get('TASK_PRESENTER_MAX_SIZE_MB', 2) if current_app and current_app.config else 0
+        max_length_bytes = max_length_mb * 1024 * 1024 # Maximum POST data size (MB)
+        if content_length and max_length_bytes and content_length > max_length_bytes:
```

Comment:
with proposed changes at line 57 ` if content_length > max_length_bytes:` would be sufficient and simplify if condition, hence no need to check `if content_length and max_length_bytes`

Reply:
Fixed via https://github.com/bloomberg/pybossa/pull/817/commits/227da7f1a58cfe1101b798586d008c5ea4ff23f0